recipe/meta.yaml

```diff
@@ -9,10 +9,17 @@ source:
     url: http://www.hdfgroup.org/ftp/HDF/HDF_Current/src/hdf-{{ version }}.tar.bz2
     sha256: 55d3a42313bda0aba7b0463687caf819a970e0ba206f5ed2c23724f80d2ae0f3
     patches:
-        - max_files.patch
-
+        # - max_files.patch
+        - patches/max_files.patch
+        - patches/0001-Don-t-build-ctest0-and-run-ncgen-when-not-testing.patch
+        - patches/debian/64bit.patch
+        # - patches/debian/hdf-4.2.10-aarch64.patch
+        - patches/debian/hdf-4.2.10-aarch64.01.patch
+        - patches/debian/hdfi.h.01.patch
+        # - patches/debian/hdfi.h.patch
```

Comment:
Why do we have multiple copies of these patches? If we are going to adopt the conda-forge packaging & patches, we should be following the names within their recipe.